### PR TITLE
[HDInsight] - Update SDK version to 5.0.0

### PR DESCRIPTION
**Pull request by idear1203**
## Swagger PR Links

- [Azure/azure-rest-api-specs#5538: [HDInsight] - Move API with version 2018-06-01-preview to stable folder](https://github.com/Azure/azure-rest-api-specs/pull/5538)

## Note

This PR aims to update SDK version from 4.4.0-preview to 5.0.0. This is the first Swagger-based HDInsight SDK with stable version number.

### CODE REVIEW

File: src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj
Comment (dsgouda) [inline src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj]: It would be helpful if you point out the features being released in this stable version in the `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
LOoks good for the most part

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

File: src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj
Comment (idear1203) [reply src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj]: Good suggestion! I have updated the `PackageReleaseNotes`

**Comment by idear1203** [comment]
Please don't merge. We need to update the SDK Release note.

**Comment by dsgouda** [comment]
@idear1203 let us know when this is ready to merge

**Comment by idear1203** [comment]
startbuild

**Comment by idear1203** [comment]
@dsgouda This PR is ready to merge.

**Pull request merged**
